utils/install-mlir.sh

```diff
@@ -1,15 +1,22 @@
-git clone https://github.com/llvm/llvm-project.git
-# Check out a specific branch that is known to work with ONNX MLIR.
-cd llvm-project && git checkout 076475713c236081a3247a53e9dbab9043c3eac2 && cd ..
-mkdir llvm-project/build
+if [ ! -d llvm-project ]; then
```

Comment:
Any reason for introducing these conditions?

Reply:
It just made it easier to keep re-running the script so it didn't error-out if the cloned repo existed (e.g. testing out a different commit id). I can remove if required.

Reply:
That would be great. In a normal use case you shouldn't have to run this script more than once.

Since you tested several commit IDs, does this mean that the current commit ID used in your patch is not the latest?

Reply:
Yeah, it was just the latest when I looked at it last week.
Let me update to the latest commit id I can and revert back the extra conditions I added. Once I've tested I'll update this PR.

Reply:
Sounds great! Thanks a lot!

Reply:
Done! Commit id changed to latest master (as of a couple hours ago), and conditions rolled back.

Reply:
@doru1004 , how do I fix the doc errors reported from the ci?

Reply:
Never mind, got it.